src/data_management/rasters/RasterForm.js

```diff
@@ -572,49 +43,35 @@ class RasterFormModel extends Component {
     this.setState({ handlingDone: true });
   }
 
-  handleClickCreateRaster() {
-    this.scrollToTop();
+  
+  onSubmit = (validatedData, currentRaster) => {
+
     this.setState({ isFetching: true, openOverlay: true });
+  
     const url = "/api/v4/rasters/";
-    const observationTypeId =
-      (this.state.observationType && this.state.observationType.code) ||
-      undefined;
-
-    const intAggregationType =
-      (this.state.aggregationType &&
-        this.aggregationTypeStringToInteger(this.state.aggregationType)) ||
-      undefined;
-
-    const isoIntervalDuration = this.intervalToISODuration(
-      this.state.temporalIntervalDays,
-      this.state.temporalIntervalHours,
-      this.state.temporalIntervalMinutes,
-      this.state.temporalIntervalSeconds
-    );
-
-    if (!this.props.currentRaster) {
+     if (!currentRaster) {
       const opts = {
         credentials: "same-origin",
         method: "POST",
         headers: { "Content-Type": "application/json" },
         body: JSON.stringify({
-          name: this.state.rasterName,
-          organisation: this.state.selectedOrganisation.uuid.replace(/-/g, ""),
-          access_modifier: this.state.accesModifier,
-          observation_type: observationTypeId, //this.state.observationType,
-          description: this.state.description,
-          supplier: this.state.supplierId && this.state.supplierId.username,
-          supplier_code: this.state.supplierCode,
-          temporal: this.state.temporalBool,
-          interval: isoIntervalDuration, //'P1D', // P1D is default, = ISO 8601 datetime for 1 day",
-          rescalable: false,
+          name: validatedData.rasterName,
+          organisation: validatedData.selectedOrganisation.replace(/-/g, ""),
+          access_modifier: validatedData.accesModifier,
```

Comment:
access ipv acces?

Reply:
done ->
changed accesModifier to accesModifier